common/tdishell

```diff
@@ -0,0 +1,40 @@
+import socket
+import re
+import argparse
+
```

Comment:
Please add a short blub of comment describing what this script achieves

Reply:
base.py and tdishell : Not using these . So deleting these